curations/nuget/nuget/-/System.Configuration.ConfigurationManager.yaml

```diff
@@ -6,3 +6,6 @@ revisions:
   4.4.0:
     licensed:
       declared: MIT
+  4.5.0:
+    licensed:
+      declared: MIT
```

Comment:
https://www.nuget.org/packages/System.Data.DataSetExtensions links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT